### ISSUE TITLE
I have 2 problems:

### ISSUE THREAD

**Issue by TrabajosCristianCeballos** [ibmtjbot/tjbot#138]
1) problem's about the audio, in the raspberry pi the audio don't working,i replace the card in the step 3.

2) I'm working with the workspace-sample.json, I imported the workspace on Watson assitant and copied the credencial and the ID in the code but, when I executed the command sudo node conversation.js, the answer's: 

16376=173, 16377=3, 16378=155, 16379=98, 16380=165, 16381=3, 16382=147, 16383=98, 16384=167, 16385=3, 16386=145, 16387=98, 16388=166, 16389=3, 16390=144, 16391=98, encoding=buffer, isBuf=true, callback=function fragmentSentCallback(err) {
        if (err) {
            if (typeof cb === 'function') {
                // pass only the first error
                cb(err);
                cb = null;
            }
            return;
        }
        ++sentFragments;
        if ((sentFragments === numFragments) && (typeof cb === 'function')) {
            cb();
        }
    }, 0=128, 1=254, 2=44, 3=72, 4=31, 5=67, 6=220, 7=236, 8=7, 9=67, 10=196, 11=236, 12=8, 13=67, 14=203, 15=236, 16=4, 17=67, 18=199, 19=236, 20=5, 21=67, 22=198, 23=236, 24=6, 25=67, 26=197, 27=236, 28=6, 29=67, 30=197, 31=236, 32=8, 33=67, 34=203, 35=236, 36=5, 37=67, 38=198, 39=236, 40=7, 41=67, 42=196, 43=236, 44=8, 45=67, 46=203, 47=236, 48=6, 49=67, 50=197, 51=236, 52=4, 53=67, 54=199, 55=236, 56=5, 57=67, 58=198, 59=236, 60=2, 61=67, 62=193, 63=236, 64=8, 65=67, 66=203, 67=236, 68=13, 69=67, 70=206, 71=236, 72=12, 73=67, 74=207, 75=236, 76=13, 77=67, 78=206, 79=236, 80=10, 81=67, 82=201, 83=236, 84=7, 85=67, 86=196, 87=236, 88=11, 89=67, 90=200, 91=236, 92=9, 93=67, 94=202, 95=236, 96=9, 97=67, 98=202, 99=236, 100=10, 101=67, 102=201, 103=236, 104=7, 105=67, 106=196, 107=236, 108=12, 109=67, 110=207, 111=236, 112=10, 113=67, 114=201, 115=236, 116=10, 117=67, 118=201, 119=236, 120=11, 121=67, 122=200, 123=236, 124=7, 125=67, 126=196, 127=236, 128=11, 129=67, 130=200, 131=236, 132=11, 133=67, 134=200, 135=236, 136=12, 137=67, 138=207, 139=236, 140=14, 141=67, 142=205, 143=236, 144=10, 145=67, 146=201, 147=236, 148=12, 149=67, 150=207, 151=236, 152=8, 153=67, 154=203, 155=236, 156=14, 157=67, 158=205, 159=236, 160=13, 161=67, 162=206, 163=236, 164=14, 165=67, 166=205, 167=236, 168=12, 169=67, 170=207, 171=236, 172=12, 173=67, 174=207, 175=236, 176=13, 177=67, 178=206, 179=236, 180=14, 181=67, 182=205, 183=236, 184=12, 185=67, 186=207, 187=236, 188=11, 189=67, 190=200, 191=236, 192=12, 193=67, 194=207, 195=236, 196=11, 197=67, 198=200, 199=236, 200=10, 201=67, 202=201, 203=236, 204=14, 205=67, 206=205, 207=236, 208=13, 209=67, 210=206, 211=236, 212=7, 213=67, 214=196, 215=236, 216=8, 217=67, 218=203, 219=236, 220=14, 221=67, 222=205, 223=236, 224=6, 225=67, 226=197, 227=236, 228=12, 229=67, 230=207, 231=236, 232=13, 233=67, 234=206, 235=236, 236=12, 237=67, 238=207, 239=236, 240=8, 241=67, 242=203, 243=236, 244=6, 245=67, 246=197, 247=236, 248=13, 249=67, 250=206, 251=236, 252=14, 253=67, 254=205, 255=236, 256=13, 257=67, 258=206, 259=236, 260=15, 261=67, 262=204, 263=236, 264=11, 265=67, 266=200, 267=236, 268=12, 269=67, 270=207, 271=236, 272=14, 273=67, 274=205, 275=236, 276=15, 277=67, 278=204, 279=236, 280=14, 281=67, 282=205, 283=236, 284=16, 285=67, 286=211, 287=236, 288=17, 289=67, 290=210, 291=236, 292=16, 293=67, 294=211, 295=236, 296=16, 297=67, 298=211, 299=236, 300=13, 301=67, 302=206, 303=236, 304=15, 305=67, 306=204, 307=236, 308=20, 309=67, 310=215, 311=236, 312=17, 313=67, 314=210, 315=236, 316=18, 317=67, 318=209, 319=236, 320=18, 321=67, 322=209, 323=236, 324=19, 325=67, 326=208, 327=236, 328=20, 329=67, 330=215, 331=236, 332=16, 333=67, 334=211, 335=236, 336=20, 337=67, 338=215, 339=236, 340=17, 341=67, 342=210, 343=236, 344=16, 345=67, 346=211, 347=236, 348=16, 349=67, 350=211, 351=236, 352=14, 353=67, 354=205, 355=236, 356=18, 357=67, 358=209, 359=236, 360=15, 361=67, 362=204, 363=236, 364=17, 365=67, 366=210, 367=236, 368=19, 369=67, 370=208, 371=236, 372=17, 373=67, 374=210, 375=236, 376=17, 377=67, 378=210, 379=236, 380=23, 381=67, 382=212, 383=236, 384=20, 385=67, 386=215, 387=236, 388=22, 389=67, 390=213, 391=236, 392=19, 393=67, 394=208, 395=236, 396=21, 397=67, 398=214, 399=236, 400=20, 401=67, 402=215, 403=236, 404=14, 405=67, 406=205, 407=236, 408=21, 409=67, 410=214, 411=236, 412=15, 413=67, 414=204, 415=236, 416=17, 417=67, 418=210, 419=236, 420=21, 421=67, 422=214, 423=236, 424=20, 425=67, 426=215, 427=236, 428=19, 429=67, 430=208, 431=236, 432=22, 433=67, 434=213, 435=236, 436=21, 437=67, 438=214, 439=236, 440=25, 441=67, 442=218, 443=236, 444=18, 445=67, 446=209, 447=236, 448=17, 449=67, 450=210, 451=236, 452=23, 453=67, 454=212, 455=236, 456=19, 457=67, 458=208, 459=236, 460=23, 461=67, 462=212, 463=236, 464=21, 465=67, 466=214, 467=236, 468=23, 469=67, 470=212, 471=236, 472=24, 473=67, 474=219, 475=236, 476=25, 477=67, 478=218, 479=236, 480=25, 481=67, 482=218, 483=236, 484=24, 485=67, 486=219, 487=236, 488=18, 489=67, 490=209, 491=236, 492=25, 493=67, 494=218, 495=236, 496=22, 497=67, 498=213, 499=236, 500=22, 501=67, 502=213, 503=236, 504=22, 505=67, 506=213, 507=236, 508=21, 509=67, 510=214, 511=236, 512=22, 513=67, 514=213, 515=236, 516=25, 517=67, 518=218, 519=236, 520=21, 521=67, 522=214, 523=236, 524=23, 525=67, 526=212, 527=236, 528=25, 529=67, 530=218, 531=236, 532=22, 533=67, 534=213, 535=236, 536=24, 537=67, 538=219, 539=236, 540=27, 541=67, 542=216, 543=236, 544=23, 545=67, 546=212, 547=236, 548=24, 549=67, 550=219, 551=236, 552=24, 553=67, 554=219, 555=236, 556=20, 557=67, 558=215, 559=236, 560=23, 561=67, 562=212, 563=236, 564=23, 565=67, 566=212, 567=236, 568=27, 569=67, 570=216, 571=236, 572=23, 573=67, 574=212, 575=236, 576=27, 577=67, 578=216, 579=236, 580=26, 581=67, 582=217, 583=236, 584=27, 585=67, 586=216, 587=236, 588=27, 589=67, 590=216, 591=236, 592=29, 593=67, 594=222, 595=236, 596=27, 597=67, 598=216, 599=236, 600=30, 601=67, 602=221, 603=236, 604=26, 605=67, 606=217, 607=236, 608=26, 609=67, 610=217, 611=236, 612=28, 613=67, 614=223, 615=236, 616=29, 617=67, 618=222, 619=236, 620=26, 621=67, 622=217, 623=236, 624=25, 625=67, 626=218, 627=236, 628=30, 629=67, 630=221, 631=236, 632=27, 633=67, 634=216, 635=236, 636=25, 637=67, 638=218, 639=236, 640=29, 641=67, 642=222, 643=236, 644=31, 645=67, 646=220, 647=236, 648=29, 649=67, 650=222, 651=236, 652=27, 653=67, 654=216, 655=236, 656=27, 657=67, 658=216, 659=236, 660=26, 661=67, 662=217, 663=236, 664=26, 665=67, 666=217, 667=236, 668=27, 669=67, 670=216, 671=236, 672=24, 673=67, 674=219, 675=236, 676=25, 677=67, 678=218, 679=236, 680=23, 681=67, 682=212, 683=236, 684=25, 685=67, 686=218, 687=236, 688=25, 689=67, 690=218, 691=236, 692=24, 693=67, 694=219, 695=236, 696=24, 697=67, 698=219, 699=236, 700=23, 701=67, 702=212, 703=236, 704=21, 705=67, 706=214, 707=236, 708=24, 709=67, 710=219, 711=236, 712=25, 713=67, 714=218, 715=236, 716=28, 717=67, 718=223, 719=236, 720=30, 721=67, 722=221, 723=236, 724=27, 725=67, 726=216, 727=236, 728=29, 729=67, 730=222, 731=236, 732=224, 733=188, 734=35, 735=19, 736=31, 737=67, 738=220, 739=236, 740=26, 741=67, 742=217, 743=236, 744=26, 745=67, 746=217, 747=236, 748=28, 749=67, 750=223, 751=236, 752=24, 753=67, 754=219, 755=236, 756=226, 757=188, 758=33, 759=19, 760=30, 761=67, 762=221, 763=236, 764=29, 765=67, 766=222, 767=236, 768=226, 769=188, 770=33, 771=19, 772=25, 773=67, 774=218, 775=236, 776=23, 777=67, 778=212, 779=236, 780=27, 781=67, 782=216, 783=236, 784=30, 785=67, 786=221, 787=236, 788=30, 789=67, 790=221, 791=236, 792=30, 793=67, 794=221, 795=236, 796=28, 797=67, 798=223, 799=236, 800=31, 801=67, 802=220, 803=236, 804=27, 805=67, 806=216, 807=236, 808=28, 809=67, 810=223, 811=236, 812=27, 813=67, 814=216, 815=236, 816=25, 817=67, 818=218, 819=236, 820=28, 821=67, 822=223, 823=236, 824=28, 825=67, 826=223, 827=236, 828=24, 829=67, 830=219, 831=236, 832=30, 833=67, 834=221, 835=236, 836=29, 837=67, 838=222, 839=236, 840=29, 841=67, 842=222, 843=236, 844=27, 845=67, 846=216, 847=236, 848=30, 849=67, 850=221, 851=236, 852=28, 853=67, 854=223, 855=236, 856=29, 857=67, 858=222, 859=236, 860=26, 861=67, 862=217, 863=236, 864=225, 865=188, 866=34, 867=19, 868=29, 869=67, 870=222, 871=236, 872=225, 873=188, 874=34, 875=19, 876=226, 877=188, 878=33, 879=19, 880=31, 881=67, 882=220, 883=236, 884=228, 885=188, 886=39, 887=19, 888=224, 889=188, 890=35, 891=19, 892=224, 893=188, 894=35, 895=19, 896=30, 897=67, 898=221, 899=236, 900=30, 901=67, 902=221, 903=236, 904=27, 905=67, 906=216, 907=236, 908=31, 909=67, 910=220, 911=236, 912=30, 913=67, 914=221, 915=236, 916=26, 917=67, 918=217, 919=236, 920=28, 921=67, 922=223, 923=236, 924=224, 925=188, 926=35, 927=19, 928=226, 929=188, 930=33, 931=19, 932=224, 933=188, 934=35, 935=19, 936=31, 937=67, 938=220, 939=236, 940=30, 941=67, 942=221, 943=236, 944=29, 945=67, 946=222, 947=236, 948=29, 949=67, 950=222, 951=236, 952=30, 953=67, 954=221, 955=236, 956=30, 957=67, 958=221, 959=236, 960=31, 961=67, 962=220, 963=236, 964=224, 965=188, 966=35, 967=19, 968=27, 969=67, 970=216, 971=236, 972=30, 973=67, 974=221, 975=236, 976=31, 977=67, 978=220, 979=236, 980=224, 981=188, 982=35, 983=19, 984=28, 985=67, 986=223, 987=236, 988=226, 989=188, 990=33, 991=19, 992=26, 993=67, 994=217, 995=236, 996=31, 997=67, 998=220, 999=236, 1000=31, 1001=67, 1002=220, 1003=236, 1004=30, 1005=67, 1006=221, 1007=236, 1008=226, 1009=188, 1010=33, 1011=19, 1012=224, 1013=188, 1014=35, 1015=19, 1016=31, 1017=67, 1018=220, 1019=236, 1020=30, 1021=67, 1022=221, 1023=236, 1024=30, 1025=67, 1026=221, 1027=236, 1028=28, 1029=67, 1030=223, 1031=236, 1032=31, 1033=67, 1034=220, 1035=236, 1036=28, 1037=67, 1038=223, 1039=236, 1040=30, 1041=67, 1042=221, 1043=236, 1044=28, 1045=67, 1046=223, 1047=236, 1048=31, 1049=67, 1050=220, 1051=236, 1052=29, 1053=67, 1054=222, 1055=236, 1056=27, 1057=67, 1058=216, 1059=236, 1060=26, 1061=67, 1062=217, 1063=236, 1064=28, 1065=67, 1066=223, 1067=236, 1068=30, 1069=67, 1070=221, 1071=236, 1072=224, 1073=188, 1074=35, 1075=19, 1076=29, 1077=67, 1078=222, 1079=236, 1080=29, 1081=67, 1082=222, 1083=236, 1084=225, 1085=188, 1086=34, 1087=19, 1088=30, 1089=67, 1090=221, 1091=236, 1092=29, 1093=67, 1094=222, 1095=236, 1096=224, 1097=188, 1098=35, 1099=19, 1100=225, 1101=188, 1102=34, 1103=19, 1104=227, 1105=188, 1106=32, 1107=19, 1108=29, 1109=67, 1110=222, 1111=236, 1112=229, 1113=188, 1114=38, 1115=19, 1116=225, 1117=188, 1118=34, 1119=19, 1120=31, 1121=67, 1122=220, 1123=236, 1124=226, 1125=188, 1126=33, 1127=19, 1128=224, 1129=188, 1130=35, 1131=19, 1132=225, 1133=188, 1134=34, 1135=19, 1136=30, 1137=67, 1138=221, 1139=236, 1140=227, 1141=188, 1142=32, 1143=19, 1144=226, 1145=188, 1146=33, 1147=19, 1148=30, 1149=67, 1150=221, 1151=236, 1152=224, 1153=188, 1154=35, 1155=19, 1156=30, 1157=67, 1158=221, 1159=236, 1160=226, 1161=188, 1162=33, 1163=19, 1164=28, 1165=67, 1166=223, 1167=236, 1168=28, 1169=67, 1170=223, 1171=236, 1172=29, 1173=67, 1174=222, 1175=236, 1176=224, 1177=188, 1178=35, 1179=19, 1180=29, 1181=67, 1182=222, 1183=236, 1184=224, 1185=188, 1186=35, 1187=19, 1188=224, 1189=188, 1190=35, 1191=19, 1192=227, 1193=188, 1194=32, 1195=19, 1196=226, 1197=188, 1198=33, 1199=19, 1200=30, 1201=67, 1202=221, 1203=236, 1204=225, 1205=188, 1206=34, 1207=19, 1208=29, 1209=67, 1210=222, 1211=236, 1212=225, 1213=188, 1214=34, 1215=19, 1216=31, 1217=67, 1218=220, 1219=236, 1220=28, 1221=67, 1222=223, 1223=236, 1224=224, 1225=188, 1226=35, 1227=19, 1228=226, 1229=188, 1230=33, 1231=19, 1232=224, 1233=188, 1234=35, 1235=19, 1236=31, 12^C2019-04-29 15:35:12 sigHandler: Unhandled signal 2, terminating

In summary, the problems're that the audio don't working and the exit's not the exit that I'm waiting

If somebody can help me, please, send me a email to cdceballor@eafit.edu.co or saturnoadv@gmail.com 

Thanks

**Comment by jweisz** [comment]
Closing due to inactivity. Please re-open if you are still having issues.
